cabal.GHC-9.4.8.config

```diff
@@ -201,7 +201,7 @@ constraints: any.BoundedChan ==1.0.3.0,
              any.os-string ==2.0.7,
              any.panic ==0.4.0.1,
              any.parallel ==3.2.2.0,
-             any.parameterized-utils ==2.1.9.0,
+             any.parameterized-utils ==2.1.10.0,
```

Comment:
I suspect that you will also need to bump the `what4` version after it is released to Hackage.